src/Common/ThreadStatus.h

```diff
@@ -243,39 +254,34 @@ class ThreadStatus : public boost::noncopyable
     ///   query.
     void setInternalThread();
 
-    /// Starts new query and create new thread group for it, current thread becomes master thread of the query
-    void initializeQuery();
-
     /// Attaches slave thread to existing thread group
-    void attachQuery(const ThreadGroupStatusPtr & thread_group_, bool check_detached = true);
+    void attachTo(const ThreadGroupStatusPtr & thread_group_, bool check_detached = true);
```

Comment:
Maybe `attachToGroup`? Or just `attachToGroup`?
To make it a bit more similar to `detachGroup`